src/test/java/uk/gov/hmcts/reform/sscs/service/BulkPrintServiceTest.java

```diff
@@ -82,6 +86,90 @@ public void willSendToBulkPrintWithAdditionalData() {
         assertEquals("letterIds must be equal", Optional.of(LETTER_ID), letterIdOptional);
     }
 
+    @Test
+    public void willSendToBulkPrintWithAdditionalDataThatIncludesRecipients() {
+        when(sendLetterApi.sendLetter(eq(AUTH_TOKEN), captor.capture())).thenReturn(new SendLetterResponse(LETTER_ID));
+        Appointee appointee = Appointee.builder()
+            .name(Name.builder().firstName("Barry").lastName("Allen").build())
+            .address(Address.builder().line1("line1").build())
+            .build();
+        SSCS_CASE_DATA.getAppeal().getAppellant().setAppointee(appointee);
```

Comment:
Not your doing but not sure why this object is capitalised...

We could Appellant appellant = SSCS_CASE_DATA.getAppeal().getAppellant() before setting the two fields here for some DRY.